openc3/lib/openc3/models/interface_model.rb

```diff
@@ -253,8 +249,7 @@ def handle_config(parser, keyword, parameters)
         @protocols << parameters.dup
 
       when 'DONT_LOG'
-        parser.verify_num_parameters(0, 0, "#{keyword}")
-        @log = false
+        Logger.warn "DONT_LOG is deprecated and does nothing."
```

Comment:
Didn't want to completely remove the keyword and have plugin installs fail. Especially since we had it in our demo.